## 04_Makefile.dpatch by Alexis Sukrieh <sukria@debian.org>
## DP: Install Perl modules in $Config{vendorlib}

--- a/Makefile
+++ b/Makefile
@@ -30,7 +30,7 @@
 
 # Overwrite that variable with the Perl vendorlib Config value if 
 # you package Backup Manager
-PERL5DIR?="$(DESTDIR)$(shell perl -MConfig -e 'print "$$Config{sitelib}"')"
+PERL5DIR?="$(DESTDIR)$(shell perl -MConfig -e 'print "$$Config{vendorlib}"')"
 
 # Some static paths, specific to backup-manager
 LIBDIR=$(DESTDIR)/usr/share/backup-manager
